cookbooks/phase_diagram.prm

```diff
@@ -0,0 +1,136 @@
+set Use years in output instead of seconds = true
+
+# set a easy solving scheme
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Number of cheap Stokes solver steps = 0
+    set Linear solver tolerance = 1.0
+  end
+end
+
+# model geometry
+subsection Geometry model
+  set Model name = box
+  subsection Box
+    set X repetitions = 50
+    set Y repetitions = 50
+    set X extent      = 800e3
+    set Y extent      = 800e3
+  end
+end
+
+# Mesh refinement specifications 
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Initial global refinement          = 0
+  set Time steps between mesh refinement = 0
+end
+
+
+subsection Gravity model
+  set Model name = vertical
+  subsection Vertical
+    set Magnitude = 10.0
+  end
+end
+
+
+# Temperature boundary and initial conditions
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators   = left, right
+  set List of model names = box
+  subsection Box
+    set Left temperature   = 273
+    set Right temperature  = 2273
+  end
+end
+
+subsection Initial temperature model
+  set Model name = function
+  subsection Function
+    set Coordinate system = cartesian
+    set Variable names = x, y
+    set Function constants = XMAX=800e3, Tl=273.0, Tr=2273
+    set Function expression = Tl * (x - XMAX)/(-XMAX) + Tr * x / XMAX
+  end
+end
+
+# uncomment this part for steinberger model and pyrolitic lookup table
+#subsection Material model
+#  set Model name = Steinberger
+#  subsection Steinberger model
+#    set Data directory                                = $ASPECT_SOURCE_DIR/data/material-model/steinberger/
+#    set Lateral viscosity file name                   = temp-viscosity-prefactor.txt
+#    set Material file names                           = pyr-ringwood88.txt
+#    set Radial viscosity file name                    = radial-visc.txt
+#
+#    set Maximum lateral viscosity variation           = 1e2
+#    set Maximum viscosity                             = 1e23
+#    set Minimum viscosity                             = 1e19
+#
+#    set Use lateral average temperature for viscosity = true
+#    set Number lateral average bands                  = 10
+#
+#    set Reference viscosity                           = 1e21
+#    set Bilinear interpolation                        = true
+#    set Latent heat                                   = false
+#  end
+#end
+
+# comment the following 3 sections for steinberg model and pyrolitic lookup table
+# fields of composition
+subsection Compositional fields
+    set Number of fields = 1
+    set Names of fields = spharz
+    set Compositional field methods = field
+end
+
+# initial composition model
+subsection Initial composition model
+    set List of model names = function
+    subsection Function
+        set Coordinate system = cartesian
+        set Function expression = 0.0
+    end
+end
+
+# value for material model
+# change thermal expansivity to 0.0
```

Comment:
Can you clarify what you mean by this comment? Do you mean "the thermal expansivity needs to be zero"? Since it is already set to 0.0